.github/workflows/test_backend_e2e_and_unit_and_qc.yml

```diff
@@ -42,5 +42,7 @@ jobs:
           pip install --upgrade wheel
           pip install --upgrade setuptools
           pip install -r requirements.txt
+          TOKEN=$(grep '^PALANTIR_ENCLAVE_AUTHENTICATION_BEARER_TOKEN=' env/.env | cut -d'=' -f2)
+          python3 -m pip install vshub_sdk --upgrade --extra-index-url "https://:${TOKEN}@unite.nih.gov/artifacts/api/repositories/ri.artifacts.main.repository.9bc9cc56-4b8c-4560-9bfb-fba8ade55246/contents/release/pypi/simple" --extra-index-url "https://:${TOKEN}@unite.nih.gov/artifacts/api/repositories/ri.foundry-sdk-asset-bundle.main.artifacts.repository/contents/release/pypi/simple"
```

Comment:
## SDK installation in GitHub action
This was the most annoying part, and made me regret opting to use the SDK.